src/test/java/org/fundacionjala/coding/manuel/CesarCryptoTest.java

```diff
@@ -0,0 +1,111 @@
+package org.fundacionjala.coding.manuel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class to test CesarCrypto.
+ */
+public class CesarCryptoTest {
+
+    /**
+     * Object of the class to test.
+     */
+    private CesarCrypto cesarCrypto;
+    private static final String VIGENERE_KEY = "LIMON";
+    private static final String TEST_WORD = "ZORRO";
+
+    /**
+     * Method to init object.
+     */
+    @Before
+    public void init() {
+        cesarCrypto = new CesarCrypto();
+    }
+
+    /**
+     * Test encode method.
+     */
+    @Test
+    public void testEncode() {
+        final int firstKey = 3;
+        final int secondKey = 5;
+        assertEquals("KROD", cesarCrypto.encode("HOLA", firstKey));
+        assertEquals("ETWWT", cesarCrypto.encode(TEST_WORD, secondKey));
+    }
+
+
+    /**
+     * Test encode methods for null and empty values.
+     */
+    @Test
+    public void testEncodeNullAndEmpty() {
+        final int key = 0;
+        assertEquals("", cesarCrypto.encode(null, key));
+        assertEquals("", cesarCrypto.encode("", key));
+    }
+
+    /**
+     * Test encode method for special characters.
+     */
+    @Test
+    public void testEncodeSpecialCharacters() {
+        final int key = 3;
+        final int secondKey = 5;
+        assertEquals("KR_OD", cesarCrypto.encode("HO_LA", key));
+        assertEquals("ETW;WT", cesarCrypto.encode("ZOR;RO", secondKey));
+    }
+
+    /**
+     * Test decode method.
+     */
+    @Test
+    public void testDecode() {
+        final int firstKey = 3;
+        final int secondKey = 5;
+        assertEquals("HOLA", cesarCrypto.decode("KROD", firstKey));
+        assertEquals("ZORRO", cesarCrypto.decode("ETWWT", secondKey));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'TEST_WORD' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)